README.md

```diff
@@ -283,6 +283,14 @@ Commands:
 
   patch <ID> <FILE>          Apply patch FILE to ID.
 
+  env-get                    Get the complete environment of the cluster master
```

Comment:
It would be possible, and perhaps more useful in this low-level tool, for env-get to take an `<ID>` (defaulting to `0`, as it does now) to get the specific env of a worker. I can see this being useful for an env browser in mesh, when it comes time to wanting to know for ***certain*** what the env of a worker is, perhaps if you suspect its not what was intended.

Though... I assume the protocol itself already supports this, right? It should be able to add a `target` qualifier to a command, and the master will forward it that specific target, rather than doing the "all" that it does now?